src/script/components/InputBar/components/InputBarControls/ControlButtons.tsx

```diff
@@ -84,21 +103,19 @@ const ControlButtons: React.FC<ControlButtonsProps> = ({
       <>
         {showFormatButton && (
           <li>
-            <FormatTextButton isActive={isFormatActive} onClick={onFormatClick} />
+            <FormatTextButton isActive={isFormatActive} isEditing={false} onClick={onFormatClick} />
```

Comment:
maybe set false to default

Reply:
asnwered [here](https://github.com/wireapp/wire-webapp/pull/18643#discussion_r1928146401), will do it